hack/test-cmd.sh

```diff
@@ -188,6 +210,22 @@ atomic-enterprise start \
   --etcd-dir="${ETCD_DATA_DIR}" \
   --images="${USE_IMAGES}"
 
+os::test::junit::declare_suite_start "cmd/validatation"
+# validate config that was generated
+os::cmd::expect_success_and_text "openshift ex validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml" 'SUCCESS'
+os::cmd::expect_success_and_text "openshift ex validate node-config ${NODE_CONFIG_DIR}/node-config.yaml" 'SUCCESS'
+# breaking the config fails the validation check
+cp ${MASTER_CONFIG_DIR}/master-config.yaml ${BASETMPDIR}/master-config-broken.yaml
+os::util::sed '7,12d' ${BASETMPDIR}/master-config-broken.yaml
+os::cmd::expect_failure_and_text "openshift ex validate master-config ${BASETMPDIR}/master-config-broken.yaml" 'ERROR'
+
+cp ${NODE_CONFIG_DIR}/node-config.yaml ${BASETMPDIR}/node-config-broken.yaml
+os::util::sed '5,10d' ${BASETMPDIR}/node-config-broken.yaml
+os::cmd::expect_failure_and_text "openshift ex validate node-config ${BASETMPDIR}/node-config-broken.yaml" 'ERROR'
+echo "validation: ok"
+os::test::junit::declare_suite_end
```

Comment:
Since in some places we do, others we don't `echo "<group test>: ok"`, how about adding those output as part of `os::test::junit::declare_suite_start` and `os::test::junit::declare_suite_end`. These know exactly what test suite your in and can output that information. One less line of typing and you have similar output both in the console and in the report, wdyt?